web/sites/default/config/field.field.node.city_code.field_documents_and_exhibits.yml

```diff
@@ -23,7 +23,7 @@ translatable: true
 default_value: {  }
 default_value_callback: ''
 settings:
-  file_directory: code
+  file_directory: citycode
```

Comment:
The actual directory going forward for code is /code/[title]/[chapter]/[section]. I'm not sure this should be changed to citycode as that was kinda a lame URL pattern from POG 😄

Reply:
I should have marked this comment as part of a review :)

Reply:
Done. This was changed back to "code."